### PR TITLE
Fix edit

### PR DESCRIPTION
**Pull request by shadowezz**
- Allow optional fields to be cleared during edit
- Rearrange fields in view panel
- Increase wrap text length for view panel

### CODE REVIEW

File: src/main/java/seedu/address/ui/DeliverableDetailsPanel.java [hunk 1]
```diff
@@ -42,7 +42,10 @@ public DeliverableDetailsPanel(Deliverable deliverable) {
         deliverable.getContacts().value.ifPresentOrElse(contacts -> {
             for (String contact : contacts.split(",")) {
                 String cleanedContact = contact.trim().replaceAll("\\s+", " ");
-                contactList.getChildren().add(new Label("- " + cleanedContact));
+                Label label = new Label("-" + cleanedContact);
```
Comment (claraadora) [inline src/main/java/seedu/address/ui/DeliverableDetailsPanel.java:45]: Do we still want the space i.e. "- "?

File: src/main/java/seedu/address/ui/MeetingDetailsPanel.java [hunk 2]
```diff
@@ -41,7 +41,10 @@ public MeetingDetailsPanel(Meeting meeting) {
         meeting.getContacts().value.ifPresentOrElse(contacts -> {
             for (String contact : contacts.split(",")) {
                 String cleanedContact = contact.trim().replaceAll("\\s+", " ");
-                contactList.getChildren().add(new Label("- " + cleanedContact));
+                Label label = new Label("-" + cleanedContact);
```
Comment (claraadora) [inline src/main/java/seedu/address/ui/MeetingDetailsPanel.java:44]: Ditto

**Review by claraadora — APPROVED** [review]
LGTM, with very minor comments

File: src/main/java/seedu/address/ui/DeliverableDetailsPanel.java (hunk 1 above)
Comment (shadowezz) [reply src/main/java/seedu/address/ui/DeliverableDetailsPanel.java:45]: Its cos i decided to wrap contacts also but with the space there the entire thing will go to next line. Do you think its ok?

**Pull request merged**